src/contracts/GPv2Settlement.sol

```diff
@@ -361,6 +368,17 @@ contract GPv2Settlement {
                 revert(add(response, 0x20), mload(response))
             }
         }
+
+        bytes4 selector;
+        if (interaction.callData.length > 4) {
```

Comment:
A function signature has size 4 bytes, so I believe equality should be included: 
```suggestion
        if (interaction.callData.length >= 4) {
```

Reply:
:see_no_evil:

Reply:
fixed and added a unit test for this.